examples/using-loadable-components/README.md

```diff
@@ -0,0 +1,87 @@
+# Gatsby + loadable-components
+
+This repo is meant to illustrate the pattern of adding loadable-components to a Gatsby project. In this case, the Header component is being code-split out while mainting SSR.
```

Comment:
Link to loadable-components here for people who don't exactly know what it is :)